### PR TITLE
[flutter_local_notifications] Issues/dismiss notifications callback

### PR DESCRIPTION
**Pull request by matanshukry**

### CODE REVIEW

File: flutter_local_notifications/example/lib/main.dart [hunk 1]
```diff
@@ -66,6 +69,11 @@ Future<void> main() async {
       debugPrint('notification payload: ' + payload);
     }
     selectNotificationSubject.add(payload);
+  }, onDismissNotification: (String payload) async {
+    if (payload != null) {
+      debugPrint('dismiss notification payload: ' + payload);
```
Comment (bennyz) [inline flutter_local_notifications/example/lib/main.dart]: no `String.format` in dart?

File: flutter_local_notifications/lib/src/typedefs.dart [hunk 2]
```diff
@@ -3,6 +3,9 @@ import 'dart:async';
 /// Signature of callback passed to [initialize] that is triggered when user taps on a notification.
 typedef SelectNotificationCallback = Future<dynamic> Function(String payload);
 
+/// Signature of callback passed to [initialize] that is trigger when user dismiss a notification
```
Comment (bennyz) [inline flutter_local_notifications/lib/src/typedefs.dart]: triggered

File: flutter_local_notifications/example/lib/main.dart (hunk 1 above)
Comment (matanshukry) [reply flutter_local_notifications/example/lib/main.dart]: Never existed. you're a witch.

File: flutter_local_notifications/lib/src/typedefs.dart (hunk 2 above)
Comment (matanshukry) [reply flutter_local_notifications/lib/src/typedefs.dart]: Fixed

**Review by bennyz — APPROVED** [review]

**Pull request merged**